pkg/util/expansion/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
```

Comment:
Also i could swear i've gotten errors on this before but it's been in flux
On Fri, May 15, 2015 at 8:31 PM Paul Morie <pmorie@gmail.com> wrote:

> @a-robinson perhaps i'm mistaken -- i'll look. Thx
> On Fri, May 15, 2015 at 8:21 PM Alex Robinson <notifications@github.com>
> wrote:
>
>> In pkg/util/expansion/doc.go
>> <https://github.com/GoogleCloudPlatform/kubernetes/pull/7936#discussion_r30455768>
>> :
>>
>> > @@ -0,0 +1,18 @@
>> > +/*
>> > +Copyright 2014 The Kubernetes Authors All rights reserved.
>>
>> Really? There's a ton of files with 2015 in their boilerplate, including
>> the boilerplate-checking script itself
>> <https://github.com/GoogleCloudPlatform/kubernetes/blob/master/hooks/boilerplate.py>
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/GoogleCloudPlatform/kubernetes/pull/7936/files#r30455768>
>> .
>>
>